layouts/fastly/helix.vcl

```diff
@@ -625,7 +625,7 @@ sub hlx_type_purge {
     + "/purge@v1"
     + "?host=" + urlencode(req.http.X-Orig-Host)
     + "&xfh="  + urlencode(req.http.X-Forwarded-Host)
-    + "&path=" + urlencode(req.http.X-Orig-Url);
+    + "&path=" + urlencode(regsuball(req.http.X-Orig-Url, "\?.*$", ""));
```

Comment:
wouldn't it be better to pass in the original url, and trim the query string in helix-purge when needed?
the outer CDN still needs the query string to flush by url, no?

Reply:
Good point.

```suggestion
    + "&path=" + urlencode(req.http.X-Orig-Url);
```